src/control/server/ctl_storage_rpc.go

```diff
@@ -966,8 +990,15 @@ func (cs *ControlService) StorageFormat(ctx context.Context, req *ctlpb.StorageF
 		skipped:     instanceSkips,
 		mdFormatted: mdFormatted,
 	}
+	if cs.srvCfg != nil && cs.srvCfg.DisableHugepages {
+		cs.log.Notice("skipping bdev format as use of hugepages disabled in config")
```

Comment:
nit - The reasoning for the skip is included in the result returned from formatNvme, right? Just wondering if we need this message at the Notice level if the user will be notified anyway.